pkgs/development/libraries/quantlib/default.nix

```diff
@@ -7,20 +7,32 @@
 
 stdenv.mkDerivation rec {
   pname = "quantlib";
-  version = "1.29";
+  version = "1.33";
 
   outputs = [ "out" "dev" ];
 
   src = fetchFromGitHub {
     owner = "lballabio";
     repo = "QuantLib";
-    rev = "QuantLib-v${version}";
-    sha256 = "sha256-TpVn3zPru/GtdNqDH45YdOkm7fkJzv/qay9SY3J6Jiw=";
+    rev = "v${version}";
+    sha256 = "sha256-j2nRm6ebf5OU6mqmcC7wQf/qlf/K9RmmCAnfT+Au8ZM=";
   };
 
   nativeBuildInputs = [ cmake ];
   buildInputs = [ boost ];
 
+  # Required by RQuantLib, may be beneficial for others too
+  postPatch = ''
+    substituteInPlace CMakeLists.txt --replace-fail \
+    "QL_HIGH_RESOLUTION_DATE \"Enable date resolution down to microseconds\" OFF" \
+    "QL_HIGH_RESOLUTION_DATE \"Enable date resolution down to microseconds\" ON"
```

Comment:
Should this be a configuration option?

Reply:
The RQuantlib package fails without it,

Reply:
We could leave it off in quantlib, and override it in RQuantLib, but that's too much fuss for something that's probably useful for the main lib as well, no?

Reply:
I don't know, I'm not a user. I'm asking in case the default is the expected behaviour. Can this use
```
cmakeFlags = [ "-DQL_HIGH_RESOLUTION_DATE=ON" ]
```
instead of patching?

Reply:
I couldn't get it to work with cmakeflags, but I can try again. It would be nicer indeed

Reply:
That worked. Let me know if you want to be maintainer instead ;-)